src/graph/viewport.ts

```diff
@@ -50,7 +50,11 @@ export class Viewport {
     this.bindHover();
 
```

Comment:
I think you can also bind resize method directly in the constructor:
```suggestion
    this.resize = this.resize.bind(this)
    this.resize();
    window.addEventListener('resize', this.resize);
```
Then you won't need to create additional method `resizeHandler`.

Reply:
Cool, thanks for that. Here it is 9f36694